usr/src/uts/common/fs/zfs/sys/zap.h

```diff
@@ -87,22 +88,17 @@ extern "C" {
 
 /*
  * Specifies matching criteria for ZAP lookups.
+ * MT_EXACT - exact match, bypasses normalization to do strcmp()
+ * MT_FIRST - Normalizes the lookup - compose/decompose and then
+ *            match the normalized string.
+ * MT_CASE - Used to keep track of whether the lookup request is
```

Comment:
On Mon, Mar 28, 2016 at 6:38 PM, Matthew Ahrens <notifications@github.com>
wrote:

> In usr/src/uts/common/fs/zfs/sys/zap.h
> <https://github.com/openzfs/openzfs/pull/77#discussion_r57645678>:
>
> > @@ -87,22 +88,17 @@ extern "C" {
> >
> >  /*
> >   * Specifies matching criteria for ZAP lookups.
> > + * MT_EXACT - exact match, bypasses normalization to do strcmp()
> > + * MT_FIRST - Normalizes the lookup - compose/decompose and then
> > + *            match the normalized string.
> > + * MT_CASE - Used to keep track of whether the lookup request is
>
> if MT_CASE is set, then it is case-sensitive?
>

Correct.  Ultimately I believe you are asking/suggesting the comment might
be improved to indicate that if it's set then that means the lookup is case
sensitive?  I'd be hapy with that change as it helps the reader understand
the code better.



> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/openzfs/openzfs/pull/77/files/713d9ff5d7adc49d9e5858a05121641625b6f8d3#r57645678>
>